setup.py

```diff
@@ -52,7 +52,8 @@
     "importlib_metadata;python_version<'3.8'",
     "azure-iot-device~=2.11",
     "tqdm~=4.62",
-    "packaging"
+    "packaging",
+    "azure-iot-hub~=2.6.0"
```

Comment:
The IoT CLI extension maintains its own data plane service sdks, adding a data plane service SDK dependency here is redundant and does not align with existing product principles. Unless this change is an assertion or discussion point that those principles should change. Happy to talk discuss more offline.

Reply:
Updated to use the CLI Data plane service SDKs. Added arguments to allow users to specify storage account and container and build SAS URL for them. SDK version upgrade to support managed identity.